lib/font/table_glyf.js

```diff
@@ -76,12 +90,21 @@ class Glyf {
 
     for (let y = 0; y < pixels.length; y++) {
       const line = pixels[y];
-      for (let x = 0; x < line.length; x++) {
-        bitStream.writeBits(line[x], bpp);
-      }
-      if (pad) {
-        for (let x = 0; x < pad; x++) {
-          bitStream.writeBits(0, bpp);
+      if (this.font.opts.stride && this.font.opts.stride !== 1) {
+        let stride = this.font.opts.stride;
+        const alignedLine = Buffer.alloc(Math.ceil(line.length / stride) * stride);
+        alignedLine.fill(Buffer.from(line), 0, line.length);
+        for (let x = 0; x < alignedLine.length; x++) {
+          bitStream.writeBits(alignedLine[x], bpp);
+        }
+      } else {
+        for (let x = 0; x < line.length; x++) {
+          bitStream.writeBits(line[x], bpp);
+        }
+        if (pad) {
```

Comment:
Can you add a function for padding handling to avoid the warning below:
> Blocks are nested too deeply (4). Maximum allowed is 3

Reply:
done